tox.ini

```diff
@@ -55,9 +55,18 @@ deps =
     online: pytest-rerunfailures
     online: pytest-timeout
 
+    # Oldest Dependacies
+    oldestdeps: sunpy<3.1
+    oldestdeps: astropy<4.3
+    oldestdeps: gwcs<0.16
+    oldestdeps: numpy<1.18
+    oldestdeps: matplotlib<3.3
+    oldestdeps: mpl_animators<1.1
```

Comment:
Should these be `==` or `>`?

Reply:
This is a little odd, but it's done with `<` so that it picks up whatever the newest bugfix release of the minimum supported minor version is. i.e. `<3.1` will pick up `3.0.2` when it's released without us having to change anything.